### DISCUSSION TITLE
Can't start Truckersmp-cli

### DISCUSSION THREAD

**Discussion by sindney1001** [truckersmp-cli/truckersmp-cli#342] [Q&A]
[Kuvakaappaus - 2024-01-03 10-32-44](https://github.com/truckersmp-cli/truckersmp-cli/assets/155527499/5840093e-2c00-4a97-ab77-36d14a211b0b)

## Information
 - OS: [e.g. Ubuntu 20.04, Arch Linux]
 - Proton or Wine version: [e.g. `Proton 5.0-7`, `Wine 5.9`]
 - truckersmp-cli version: [e.g. `0.1.0`]

**Comment by Lucki** [comment]
We do not recommend installing python packages system-wide and your pip thinks the same. See the red error message

https://github.com/truckersmp-cli/truckersmp-cli#install
